CMakeLists.txt

```diff
@@ -1,15 +1,44 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.3)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
+
+INCLUDE(GNUInstallDirs)
```

Comment:
I see that LLVM project uses lower case for cmake commands.
I think it make sense to follow LLVM notation in case we eventually contribute this project to LLVM.